include/d/actor/d_a_majuu_flag.h

```diff
@@ -55,14 +55,14 @@ class daMajuu_Flag_packet_c : public J3DPacket {
 
 class daMajuu_Flag_c : public fopAc_ac_c {
 public:
-    void setBaseScale(float) {}
-    void setOtherMatrix(Mtx*) {}
-    void setOtherOffset(cXyz*) {}
+    void setBaseScale(float scale) { mFlagScale = scale; }
+    void setOtherMatrix(Mtx* mtx) { mpParentMtx = *mtx; }
+    void setOtherOffset(cXyz* pos) { mpParentPos = pos; }
 
 public:
     /* 0x290 */ u8 m290[0x2A0 - 0x290];
     /* 0x2A0 */ daMajuu_Flag_packet_c mPacket;
-    /* 0xA3D */ u8 mA3D[0xA40 - 0xA3D];
+    /* 0x2A4 */ u8 mA3D[0xA40 - 0x2A4];
```

Comment:
This change seems to be causing several regressions. Mistake?

Reply:
Not a mistake, it was wrong in the undecomped file because daMajuu_Flag_packet_c  was empty. Now that you merged majuu flag it's wrong to keep it though.